daemons/fenced/fenced_remote.c

```diff
@@ -715,8 +715,9 @@ remote_op_timeout(gpointer userdata)
                   CRM_XS " id=%.8s",
                   op->action, op->target, op->client_name, op->id);
     } else {
-        finalize_timed_out_op(userdata, "Fencing could not be completed "
-                                        "within overall timeout");
+        finalize_timed_out_op(userdata, "Fencing did not complete within "
+                                        "configured timeouts and retries "
+                                        "for any relevant devices");
```

Comment:
that sounds somehow confusing to me.
maybe "on any of the applicable devices" if that is what is still what is meant.
or maybe "using" instead of "on" ...

Reply:
The problem (for explaining this to the user) is that the timeout actually used for a fencing action is not a single configuration option anywhere, but the total of ((configured timeout times configured retries)) for all devices attempted (probably plus other factors like query timeouts etc.).

The goal is to give the user a hint as to what configuration is relevant.

Maybe something like: "Fencing did not complete within a total timeout based on the configured timeout and retries for any devices attempted"

Reply:
Guess the new suggestion sums the situation up in a quite comprehensible manner.

Reply:
updated commit